src/mame/drivers/missb2.cpp

```diff
@@ -440,27 +440,24 @@ MACHINE_START_MEMBER(missb2_state,missb2)
 	m_gfxdecode->gfx(1)->set_palette(*m_bgpalette);
 
 	save_item(NAME(m_sound_nmi_enable));
-	save_item(NAME(m_pending_nmi));
-	save_item(NAME(m_sound_status));
 	save_item(NAME(m_video_enable));
 }
 
 MACHINE_RESET_MEMBER(missb2_state,missb2)
 {
+	MACHINE_RESET_CALL_MEMBER(common);
```

Comment:
bublbobl.cpp already does it the 'call each other' way, I was following the convention there.